src/test/ui/suggestions/issue-59819.rs

```diff
@@ -0,0 +1,33 @@
+#![allow(warnings)]
```

Comment:
Mark the test `// run-rustfix` to make sure we don't start giving incorrect suggestions down the line.

Reply:
Fixed. I had this originally but removed it when I added the test case to make sure the suggestion wouldn't trigger for `String`, not realising that had a suggestion that meant rustfix would still work.